java/kafka/streams/src/main/java/KafkaStreamsExample.java

```diff
@@ -46,8 +46,7 @@ public static void main(String[] args) {
                 KafkaClientSupplier supplier = new TracingKafkaClientSupplier();
                 streams = new KafkaStreams(builder.build(), props, supplier);
             } else {
-                log.error("Error: TRACING_SYSTEM {} is not recognized or supported!", config.getTracingSystem());
-                streams = new KafkaStreams(builder.build(), props);
+                throw new RuntimeException("Error: TRACING_SYSTEM jaeger is not recognized or supported!");
```

Comment:
because it's in the `else` you are not saying just that "jaeger" is not support, you are saying that everything else different from "opentelemetry" is not supported, so you should change the error message accordingly. Something that shows what's the content of `tracingSystem` variable and saying it's not recognized or supported.

Reply:
@ppatierno  Refactord